styles/prosilver/template/idea_body.html

```diff
@@ -50,86 +54,86 @@
+				<input type="text" id="rfceditinput" value="{IDEA_RFC}" data-l-err="{L_ERROR}" data-l-msg="{L_RFC_ERROR}" />
+				<!-- ENDIF -->
+			</dd>
 			<!-- ENDIF -->
-		</dd>
-		<!-- ENDIF -->
-        <!-- IF IDEA_TICKET or CAN_EDIT -->
-        <dt style="width: 15%">{L_TICKET}:</dt>
-        <dd>
-
-            <a id="ticketlink" <!-- IF IDEA_TICKET -->href="https://tracker.phpbb.com/browse/PHPBB3-{IDEA_TICKET}">PHPBB3-{IDEA_TICKET}<!-- ELSE -->style="display:none"><!-- ENDIF --></a>
-
-            <!-- IF CAN_EDIT -->
-            <a href="{U_EDIT_TICKET}" id="ticketedit" data-l-add="{L_ADD}" data-l-edit="{L_EDIT}"><!-- IF IDEA_TICKET -->{L_EDIT}<!-- ELSE -->{L_ADD}<!-- ENDIF --></a>
-            <input type="text" id="ticketeditinput"<!-- IF IDEA_TICKET --> value="PHPBB3-{IDEA_TICKET}"<!-- ENDIF --> placeholder="PHPBB3-#####" data-l-err="{L_ERROR}" data-l-msg="{L_TICKET_ERROR}" />
-            <!-- ENDIF -->
-        </dd>
-        <!-- ENDIF -->
-        <!-- IF IDEA_DUPLICATE or IS_MOD -->
-        <dt style="width: 15%" class="duplicatetoggle">{L_DUPLICATE}:</dt>
-        <dd class="duplicatetoggle">
-
-            <a id="duplicatelink" <!-- IF IDEA_DUPLICATE -->href="idea.php?id={IDEA_DUPLICATE}">idea.php?id={IDEA_DUPLICATE}<!-- ELSE -->style="display:none"><!-- ENDIF --></a>
-
-            <!-- IF IS_MOD -->
-            <a href="{U_EDIT_DUPLICATE}" id="duplicateedit">{L_SET}</a>
-            <input type="text" id="duplicateeditinput"<!-- IF IDEA_DUPLICATE --> value="{IDEA_DUPLICATE}"<!-- ENDIF --> placeholder="###" data-l-err="{L_ERROR}" data-l-msg="{L_TICKET_ERROR_DUP}" />
-            <!-- ENDIF -->
-        </dd>
-        <!-- ENDIF -->
-		<!-- IF U_DELETE_IDEA or CAN_EDIT -->
-		<dt style="width: 15%">{L_MOD_IDEA}:</dt>
-		<dd>
-			<!-- IF U_DELETE_IDEA --><a href="{U_DELETE_IDEA}" data-ajax="true" data-refresh="true">{L_DELETE}</a> &bull; <!-- ENDIF -->
-            <!-- IF CAN_EDIT --><a href="{U_EDIT_TITLE}" id="titleedit">{L_TITLE_EDIT}</a> &bull; <!-- ENDIF -->
-            <a href="{U_IDEA_TOPIC}">{L_IDEA_TOPIC}</a>
-		</dd>
-		<!-- ENDIF -->
-	</dl>
+			<!-- IF IDEA_TICKET or CAN_EDIT -->
+			<dt style="width: 15%">{L_TICKET}:</dt>
+			<dd>
+
+				<a id="ticketlink" <!-- IF IDEA_TICKET -->href="https://tracker.phpbb.com/browse/PHPBB3-{IDEA_TICKET}">PHPBB3-{IDEA_TICKET}<!-- ELSE -->style="display:none"><!-- ENDIF --></a>
+
+				<!-- IF CAN_EDIT -->
+				<a href="{U_EDIT_TICKET}" id="ticketedit" data-l-add="{L_ADD}" data-l-edit="{L_EDIT}"><!-- IF IDEA_TICKET -->{L_EDIT}<!-- ELSE -->{L_ADD}<!-- ENDIF --></a>
+				<input type="text" id="ticketeditinput"<!-- IF IDEA_TICKET --> value="PHPBB3-{IDEA_TICKET}"<!-- ENDIF --> placeholder="PHPBB3-#####" data-l-err="{L_ERROR}" data-l-msg="{L_TICKET_ERROR}" />
+				<!-- ENDIF -->
+			</dd>
+			<!-- ENDIF -->
+			<!-- IF IDEA_DUPLICATE or IS_MOD -->
```

Comment:
We use ```S_``` prefix for switches, don't know how much does it matter here and in other places.